### DISCUSSION TITLE
Configure repository to receive signed commit only

### DISCUSSION THREAD

**Discussion by LuanCFBludata** [community/community#29064] [Other Feature Feedback, Questions, & Ideas]
Hello world!

I would like to configure my repository to accept commits from users who have GPG signature configured. It's possible?

**Comment by GabyCT** [comment]
Yes, it is possible for more information you can take a look at https://docs.github.com/es/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository and https://docs.github.com/es/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Reply by LuanCFBludata** [reply]
Thanks for help. This functionality works for web commits. I would like this functionality to be available in terminal commits. It's possible?

**Reply by GabyCT** [reply]
I do not think is possible, you can only sign your commits locally (in your terminal see https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

**Comment by rsclarke** [comment]
Create a branch protection rule and select **Require signed commits**.

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

**Reply by LuanCFBludata** [reply]
Thanks for help. I will check it out.
